src/WPF/WPF.Viewer/Samples/Layers/AddVectorTiledLayerFromCustomStyle/AddVectorTiledLayerFromCustomStyle.xaml.cs

```diff
@@ -0,0 +1,134 @@
+
+            // Store a list of tiled layers in the different styles.
+            foreach (string item in _itemIDs)
+            {
+                PortalItem websceneItem = await PortalItem.CreateAsync(portal, item);
+                ArcGISVectorTiledLayer vectorTiledLayer = new ArcGISVectorTiledLayer(websceneItem);
+                _vectorTiledLayers.Add(vectorTiledLayer);
+            }
+
+            // Create a map with the default style.
+            MyMapView.Map = new Map(SpatialReferences.WebMercator);
+            MyMapView.Map.Basemap.BaseLayers.Add(_vectorTiledLayers[0]);
+
+            await MyMapView.SetViewpointAsync(_defaultViewpoint);
+        }
+
+        private async void StyleChooserSelectionChanged(object sender, System.Windows.Controls.SelectionChangedEventArgs e)
+        {
+            // Determine the style from the combobox.
+            ComboBox styleChooser = sender as ComboBox;
+            ArcGISVectorTiledLayer vectorTiledLayer = _vectorTiledLayers[styleChooser.SelectedIndex];
+
+            // Clear the map of the currently loaded layer.
+            MyMapView.Map.Basemap.BaseLayers.RemoveAt(0);
+
+            if (styleChooser.SelectedIndex <= 3)
+            {
+                // No need to export - change the basemap style.
+                await MyMapView.SetViewpointAsync(_defaultViewpoint);
+            }
+            else
+            {
+                // Export a custom style.
+                await MyMapView.SetViewpointAsync(_dodgeCityViewpoint);
+                Export(vectorTiledLayer);
+            }
+            MyMapView.Map.Basemap.BaseLayers.Add(vectorTiledLayer);
+        }
+
+        private async void Export(ArcGISVectorTiledLayer vectorTiledLayer)
+        {
+            // Create the task.
+            ExportVectorTilesTask exportTask = await ExportVectorTilesTask.CreateAsync(vectorTiledLayer.Source);
+
+            ExportVectorTilesParameters parameters = await exportTask.CreateDefaultExportVectorTilesParametersAsync(MyMapView.GetCurrentViewpoint(ViewpointType.BoundingGeometry).TargetGeometry, MyMapView.MapScale * 0.1);
+
+            // By using the UseReducedFontService download option the file download speed is reduced.
+            // This limits vector tiled layers character sets and may not be suitable in every use case.
+            parameters.EsriVectorTilesDownloadOption = EsriVectorTilesDownloadOption.UseReducedFontsService;
+
+            // Get the tile cache path and item resource path for the base layer styling.
+            string tilePath = Path.Combine(Environment.ExpandEnvironmentVariables("%TEMP%"), Path.GetTempFileName() + ".vtpk");
+            string itemResourcePath = Path.Combine(Environment.ExpandEnvironmentVariables("%TEMP%"), Path.GetTempFileName() + "_styleItemResources");
+
+            // Create the export job and start it.
+            ExportVectorTilesJob _job = exportTask.ExportVectorTiles(parameters, tilePath, itemResourcePath);
```

Comment:
```suggestion
            ExportVectorTilesJob job = exportTask.ExportVectorTiles(parameters, tilePath, itemResourcePath);
```
Local variable should not have underscore at start of name.